src/winetricks

```diff
@@ -12292,6 +12292,97 @@ load_pngfilt()
 
 #----------------------------------------------------------------
 
+w_metadata powershell_core apps \
+    title="PowerShell Core" \
+    publisher="Microsoft" \
+    year="2024" \
+    media="download" \
+    file1="PowerShell-7.2.19-win-x86.msi" \
+    file2="PowerShell-7.2.19-win-x64.msi"
+    
```

Comment:
There's trailing whitespace here.

Reply:
Fixed as of https://github.com/Winetricks/winetricks/pull/2211/commits/1d323f605c4039dd2a120530c2d22926b038a52c